code/modules/vehicles/scooter.dm

```diff
@@ -85,7 +89,17 @@
 
 /obj/vehicle/ridden/scooter/skateboard/generate_actions()
 	. = ..()
+	/* honk start -- old action initialization removed to allow for suicide ollie
 	initialize_controller_action_type(/datum/action/vehicle/ridden/scooter/skateboard/ollie, VEHICLE_CONTROL_DRIVE)
+	honk end */
+	// honk start -- adds a suicide for skateboards
+	if(suicide)
+		autogrant_actions_controller.Cut()
+		initialize_controller_action_type(/datum/action/vehicle/ridden/scooter/skateboard/ollie/suicide, VEHICLE_CONTROL_DRIVE)
+	else
+		autogrant_actions_controller.Cut()
+		initialize_controller_action_type(/datum/action/vehicle/ridden/scooter/skateboard/ollie, VEHICLE_CONTROL_DRIVE)
+	// honk end
```

Comment:
Copied from the requested changed in #120
Instead of commenting out the old version and then adding the new version after, change it in-place and comment the lines changed.

Reply:
So for this one the previous code isn't really removed but is surrounded by additional code. How do you want me to communicate that?

Reply:
It's part of the change, so you don't need that extra honk comment. The extra information in the comment is useful, but it'd be better suited to a regular comment.